### DISCUSSION TITLE
Stuck on Chinese voice

### DISCUSSION THREAD

**Discussion by iamnikv** [Hypfer/Valetudo#1876] [Q&A / Support] [answered]
After resetting my vacuum to it's default firmware (Roborock S50) the language changed back to Chinese from English. After installing Valetudo I did not find a clear guide on changing the voice pack. Could anyone help with this?

**Comment by sky321** [comment]
see  #1772

**Reply by iamnikv** [reply]
![image](https://github.com/Hypfer/Valetudo/assets/12248943/e0b3167a-8aa0-49c6-aa43-9b1b2b708e4a)
![image](https://github.com/Hypfer/Valetudo/assets/12248943/885acfd7-ed87-470f-a9a6-558792051ab0)

**Reply by iamnikv** [reply]
In https://github.com/Hypfer/Valetudo/discussions/1772 they say to put in "any URL and any hash".. what does that mean?

**Comment by iamnikv** [comment, marked as answer]
Found the answer here: https://www.ajfriesen.com/cloud-free-robot-vacuum/#basic-setup

Language packs: [https://builder.dontvacuum.me/pkg/voice/rockrobo/voice-v1/](https://builder.dontvacuum.me/pkg/voice/rockrobo/voice-v1/?ref=ajfriesen.com)
Hashes: [https://builder.dontvacuum.me/pkg/voice/rockrobo/voice-v1/pkgs.md5](https://builder.dontvacuum.me/pkg/voice/rockrobo/voice-v1/pkgs.md5?ref=ajfriesen.com)

For English enter the following in the fields:
URL: https://builder.dontvacuum.me/pkg/voice/rockrobo/voice-v1/english.pkg
Language code: EN
Hash: 911bdd115c5b93823e6a87e6c80633cc

**Reply by iamnikv** [reply]
Updated to firmware version 2034 and latest Valetudo version 2023.10.0 and the above stopped working. 
Here is what works with those versions:

For English enter the following in the fields:
URL: https://builder.dontvacuum.me/pkg/voice/rockrobo/voice-v1/english.pkg
Language code: **en**
Hash: 911bdd115c5b93823e6a87e6c80633cc

Yep, 'EN' has to be lowercased..
